docker/cytoscape/supervisor/supervisord.conf

```diff
@@ -0,0 +1,19 @@
+[supervisord]
+nodaemon=false
+
+[program:cytoscape]
+command=/bin/bash -c 'rm -rf /root/CytoscapeConfiguration && xvfb-run --auth-file /root/.Xauth -s "-screen 0 1920x1080x24" /cytoscape-unix-3.8.2/cytoscape.sh'
+priority=1
+stdout_logfile=/dev/fd/1
+stdout_logfile_maxbytes=0
+
+[program:x11vnc]
+command=/bin/bash -c 'x11vnc -display :99 -auth /root/.Xauth'
```

Comment:
What does this do?  Is it needed if we aren't trying to use the Cytoscape graphics?

Reply:
The Graphics machine is still running in the background in a headless mode, I believe that this command is still required to launch cytoscape even if we don't use the x11vnc software directly.